### ISSUE TITLE
Qestion for help straight for @Johndrinkwater

### ISSUE THREAD

**Issue by GTPunk** [johndrinkwater/ps-home-archive#1]
so,i want to export all models and textures for example from basic_apartment_evening_T037.sdat (harbour studio). What i need to do? i googled how to decrypt sdat to dat but what to do next? contentID and what-not,im not familiar with ps3 things.can you help me?

**Comment by johndrinkwater** [comment]
`sdat` files are encrypted with the PS3 platform encryption scheme… which is the reason I have not done any work on the files 😐

**Comment by GTPunk** [comment]
@johndrinkwater but can you create any easy tool for decrypt/view/export files from sdat's? like, you know, there is wrestling modmakers who learn any aspects of wwe games, even in old-gen games they learn what files inside, how to export from them, how to replace models, how to put files back inside. gigantic job.can you at least ask for colaboration with any ps3 hackers?i want to create ps home mod for another game

**Comment by GTPunk** [comment]
@johndrinkwater as far i know it needs to decrypt sdat to dat,then crypt dat to edat-convert it to pkg.or export from edat.problem is - to crypt to edat we need to know a lot of info,like cID,klak file or how it called ,ect ect. this is how one of edat tool decrypts sdat 
[Архив ZIP - WinRAR.zip](https://github.com/johndrinkwater/ps-home-archive/files/1628851/ZIP.-.WinRAR.zip)
also try to learn code of pshome's pkg (the app itself) maybe there something userful

**Comment by snakescater** [comment]
Is there any new info on exporting models from these .sdat files? Iv gotten as far as converting to .dat, is there a way to extract the files from within the bin of the .dat?

**Comment by MercuryValentine** [comment]
.dat files contain ambiguous data. From Looking at basic_apartment_evening (Harbour Studio) and a couple other random files I have, they both appear to be .bar/.sharc files, which you can open with unbar or [offzip](http://aluigi.altervista.org/mytoolz.htm). You can read more about them [here](https://zenhax.com/viewtopic.php?t=63). I've yet to work on opening or modifying the files since the preservation of missing game data is currently more important, but it'll probably require some reverse engineering and programming to extract everything from the .dat files. 

Also, anyone wanting to use the assets for other purposes should try using unbar/offzip on older versions of the game, such as 0.8.3 (I'm missing some versions, if anyone has new links that aren't public, or just game data in general). The closed beta has the Harbour Studio in its PKG for example, so give that a go.

**Comment by GTPunk** [comment]
@MercuryValentine wow i've just a few days ago found about unbar and downloaded 0.8.3 shocking... i've found it on ps3hax net or something like that. but it seems doesnt work for me since its for win xp.

**Comment by MercuryValentine** [comment]
I don't recall if it worked for me (Windows 7), however offzip is pretty simple if you know how to use the command line, and extracted the files fine.

**Comment by GTPunk** [comment]
@MercuryValentine 
 - open input file:    C:\Program Files (x86)\rpcs3\dev_hdd0\game\NPIA00005\USRDIR\SCENE_APARTMENT.BAR
- zip data to check:  32 bytes
- zip windowBits:     15
- seek offset:        0x00000000  (0)
- open output file:   SCENE_APARTMENT_unpack.BAR
.
- zlib Z_DATA_ERROR, the data in the file is not in zip format
  or uses a different windowBits value (-z). Try to use -z -15

- 2 bytes read (zipped)
- 0 bytes unzipped
in other words, it was not exported. it created only _unpack.bar version

**Comment by MercuryValentine** [comment]
You have to run the command flags which were specified in the above link and the error it returned: offzip -a -z -15 SCENE_APARTMENT.BAR

Hope that works.

**Comment by GTPunk** [comment]
@MercuryValentine Error: recheck your options, -15 is not valid or some are missing
C:\Users\днс>"C:\Program Files (x86)\rpcs3\dev_hdd0\game\NPIA00005\USRDIR\offzip.exe" "C:\Program Files (x86)\rpcs3\dev_hdd0\game\NPIA00005\USRDIR\SCENE_APARTMENT.BAR" offzip -a -z -15 SCENE_APARTMENT.BAR

**Comment by MercuryValentine** [comment]
I extracted and uploaded them for you instead here (edit: removed due to temporary file upload expiration). If you'd like to extract other files yourself, you can read the [usage guide](http://zenhax.com/viewtopic.php?t=5)

**Comment by GTPunk** [comment]
@MercuryValentine which files are models? .dat? and how to import them, any 3d max script?

**Comment by RedFox102** [comment]
I'm also looking to extract some models, mostly from scenes, harbour studio being at the top of the list, along with central plaza. Also the Café, but that seems to be missing, and I likely would have had it if my original hard drive hadn't died, I had been on since 2009 and I'm sure I visited every space possible.

I'm mostly just chiming in to show that another person is still interested in this. There seems to be some useful information here, so I'll go over it and see if I get anywhere.

**Comment by GTPunk** [comment]
@RedFox102 if we fail, there are also lkwd maps in avakin life (unity), atom universe (UE 4), harbor recreated in GMod (right now i'm trying to make it look right with gta 5 textures)
Also to what we can port it : vrchat, almost the same sinespace, make fully look-a-like gta 5 mod

**Comment by josephlee222** [comment]
If someone is recreating pshome, I will suggest that the UI for chat dialogs and menu screens to be a flat, colourful design perhaps like material instead of focusing on something futuristic to re-create the feeling of pshome

**Comment by MercuryValentine** [comment]
@RedFox102 Whether or not you had it depended on how recently you visited it; the game only reserved  3, 7, or 12GB of space for data, after which it started deleting things, so don't feel too bad about it. If you ever figure out anything useful, PM me. I'm interested in eventually making a server emulator, so any help regarding file formats and what not is appreciated. 

@GTPunk I haven't got into opening the models, though I've opened the texture images with GIMP. You can try opening the .dat files in 3DS MAX and Maya (which I know they used for asset modeling at the time), or Blender.

**Comment by GTPunk** [comment]
@MercuryValentine in .xml said theres .mdl files. creator of unbar said in read-me that there's havoc model files. can you try offzip any .dat?

**Comment by MercuryValentine** [comment]
@GTPunk Those .dat files could require offzip, but they could also simply be model files, so I don't know.  You'll have to just try opening them and see what what works at this point, or find a program that detects what an unknown file is.

**Comment by GTPunk** [comment]
@MercuryValentine that is the point: offzip is often show errors for me. can you try?

**Comment by snakescater** [comment]
Getting to the .day is as far as I get. Sony has an internal program that reads that file type it seems. IV tried for awhile and could not figure it out. IV ran it by some buddy's that major in networking and still an unopenable file

**Comment by snakescater** [comment]
Best bet would be find a past developer for psh for this to happen. But I doubt a nda agreement would leave that out.

**Comment by MercuryValentine** [comment]
I ran the files through some programs and checked the file signatures, and none of them matched any formats I could find, and given snakescater's efforts, means it's most likely a proprietary format that only the HDK (Home Development Kit) can open, which nobody has uploaded anywhere at this point. 00001e22.dat, 006d6a42.dat, and 00001712.dat  are all the same filetypes, which you can try running through some forums for some help on reverse engineering the files, however you will undoubtedly need to program your own file extraction program. I personally might consider writing one in the distant future, but it's low on my priority list.

Edit: Forgot to mention that the 3 files I listed are most likely the models. They contain plaintext of directories to the .dds textures, whereas the others reference things such as the sun, area lights, etc..

**Comment by snakescater** [comment]
If you ever decide to proceed with the program let me know, I'd be interested in helping. I have some contacts that may be able to help solve some of the issues.

**Comment by GTPunk** [comment]
@MercuryValentine OR, we can try to emulate server for 0.8.3. on rcps3. I tryed to run it and it says in console that some files are missing (folders CLANS, WORLDMAP ect.) and later said that it's asking for internet connection. And, if we can run at least just map thru emulation, we can try to use NinjaRipper or 3dripperdx on it

**Comment by RedFox102** [comment]
@GTPunk That's what I was thinking, I think we would have better luck on actual hardware though.

I have a ps3 on cfw, probably at least a few of you do too, does anyone know what was possible with the home app in debug mode? I remember seeing a video of someone accidentally going into debug mode after the server was taken down for maintenance, but they didn't show all the options. What if there was an option to load a scene on your hard drive without going online? We probably wouldn't be so lucky, but if we could manage to load a scene, a memory dump at that point would probably provide some useful information, right?

**Comment by snakescater** [comment]
I have a DEX console somewhere in storage, But I never played psh on it so mine does not have it installed nor any downloaded data. I do have a console that I spent some time in game with so i'm sure its content rich.

**Comment by MercuryValentine** [comment]
@snakescater Thanks for the offer. I'll remember if I do it. What model is your PS3 with Home installed? Given how much data is missing, there's a very high chance it has data I don't have, even if only 3GB was reserved, particularly if it wasn't used later on in Home's lifespan. If you don't want to jailbreak it, I'd even be willing to buy it off of you. EDIT: On second thought, you can simply use the Data Transfer Utility to copy Home from your OFW PS3 to your CFW PS3, which would allow you to copy it to a USB, no extra jailbreak required.


@GTPunk rpcs3 doesn't emulate the PS3's WiFi card as of yet. It's a planned feature for the long-term, however in terms of emulating the server, it'd only be a small workflow speed increase over just running Home on a PS3 for as long as it was useful.

@RedFox102 There was indeed an offline mode for the game, which I'll be trying to find by modifying the game's EBOOT soon enough, but I don't know if it'll be there, if it's a controller button combination in the menu, or if it's not even part of the EBOOT, though. As for other features, you can find a few videos on YouTube, it's mostly Moderator stuff; turn invisible, teleport, view scene list, etc.. The Options menu on the login screen is only for language/region. And if you mean recording a memory dump as the PS3 opens the proprietary .dat files for reverse engineering them, I presume it'd be useful, yeah.

**Comment by RedFox102** [comment]
Honestly don't know what I'm doing, but I messed around a little bit with a developer eboot I found earlier. I threw it on my ps3 out of curiousity and tried to activate the console with a netcheat table known to work with the same version, and unsurprisingly it didn't work from the login screen. There seemed to be a command to view game models, so I wanted to at least see what happened.

I also decrypted the eboot and had a look at the elf in a hex editor, and it does mention an "offline mode" a couple times, once preceded by "[CROSS] [TRIANGLE] [CIRCLE] [SQUARE] [R1]", and once followed by a check for an sdc file. Then I checked the non developer version, and it also mentions "offline mode", but only once with the button combination, if that's even what it is.

I guess that was fun even if I didn't accomplish anything.

**Comment by MercuryValentine** [comment]
@RedFox102 Nice to know all that ahead of time, I'll definitely try hammering away at that then, and thanks for the effort.

**Comment by RedFox102** [comment]
Do any of you know where I could find a collection of old packages for home? I'm looking for a version around 1.60, which I think would be right before they did away with the psp-like device and added the new navigator most of us are probably familiar with.

Honestly I just want to poke around it for fun, I doubt I'll actually accomplish anything. I did look at a very old version, I believe it was 0.98 and at that time it seems that the offline mode was referred to as "offline test mode". 

It also had some interesting information such as the build date, which was sometime in November 2007, and what could have been some short notes between developers, though I'm not sure why they would be in the application, but I did see something about an object that needed to be textured or retextured.

**Comment by MercuryValentine** [comment]
I've been meaning to organize and upload my PKG collection to Dropbox for some friends, so I'll update here when it's done. I've spent hours collecting all these from what I could find on the internet. Sadly some of them like 0.7.3 and 0.8.6 had dead links from Mediafire and such no matter where I looked, but I have about ~50% of them.

**Comment by GTPunk** [comment]
edit: oh, no, its just me being stupid,sorry,solved:-)
edit 2: this is my recreation of harbour studio, ready for vrchat. https://yadi.sk/d/HTrzrGLB3WMC99 download button is "скачать". i can't upload it to vrchat by myself somehow, maybe its because some amazon servers are blocked in russia, idk. it's a mix of gm_personalspace reworked, original and gta 5 textures. try to upload it to vrchat by yourself.

**Comment by MercuryValentine** [comment]
@RedFox102 [Here](https://www.dropbox.com/sh/djin7d0fe7300p9/AACId9shg5Eztd_IneXhYTNia?dl=0) they are at last. Enjoy.

@GTPunk I don't have VRChat so I can't play it, but nice work.

**Comment by RedFox102** [comment]
@MercuryValentine Thanks for this!

**Comment by RedFox102** [comment]
Also, this video is gold: http://www.gdcvault.com/play/1013809/-SPONSORED-PS-Home-Dev)

I've seen a few other videos from that conference, but I had no idea this one existed. He mentions the developer version of the home application, and also says that it runs offline, and the way he says it makes it seem like it's designed to only run offline. A couple in-game videos also show the game seemingly running in offline mode, which was pretty cool to see. I wonder if the leaked developer eboot I've been messing with is really the same as what he's using, aside from his version obviously being much older.

I found a post from a couple years ago from someone who bought a couple old reference units, and apparently one of them contained playstation home developer builds. I would really like to get in touch with them somehow, but I've not been approved on the forum I found them on. I'm crossing my fingers that they still have the files and are willing to share them, just imagine the treasure trove of information this unit could contain if it belonged to a developer for playstation home.

**Comment by MercuryValentine** [comment]
@RedFox102 Do you still have the link or know which forum it was on? I might be able to contact them.

**Comment by RedFox102** [comment]
@MercuryValentine Sure thing, I found the post here: https://assemblergames.com/threads/decr-1000a-4-53-firmware.62919/#post-902502

**Comment by MercuryValentine** [comment]
@RedFox102 I was hoping not, but I have in fact already contacted and gotten the files from him, and he said he'd contact me if he found anything new on the second one. Aside from having the developer EBOOT, which is as far as I know is identical to the leaked one you found (version aside - 1.41), there's sadly not much that's exciting. It had zero items or spaces downloaded except the defaults, some commandline history, etc.. I didn't include it in the Dropbox link because I'm still working on acquiring more copies of the dev version, but I'll make sure to upload it all once I get everything I can of it.

**Comment by RedFox102** [comment]
@MercuryValentine Ah, well that's a bit disappointing. Look forward to that upload though.

**Comment by RedFox102** [comment]
Anyone know what the devconfig.txt in the dev version actually does? It doesn't come with a list of options or anything so I don't know.

**Comment by MercuryValentine** [comment]
From mine-
CONFIGVERS=1
COMPANYENV=Env_FuturLabLimited_1ac33256-e7ee-463c-8240-5178713d878a/
ENV=cdevc


Not clue what any of that's for at this point.

**Comment by RedFox102** [comment]
Thanks for that, I thought the file was read before the developer screen was loaded, but upon further inspection, it looks like the file isn't even touched until you hit "go online". Maybe if certain conditions are met, more options would be shown on the developer screen? Then again, you can easily find the options in plain text, and I didn't see anything.

**Comment by GTPunk** [comment]
is someone of you posted that https://assemblergames.com/threads/playstation-home-preservation.69185/ ?  if no, it would be good if our and their info mixed up
also i can approve that older versions works on rpcs3. on lowest settings, just press x and there are home beta logo and sound

**Comment by RedFox102** [comment]
@GTPunk I was wondering that too, I figured it had to be one of you guys. Not mine though.

Also, I noticed that on rpcs3 too, there's just nothing you can do with it.

Personally, I just want to mess around with the offline mode for nostalgia reasons, and if we can get those scene files converted into another format, I want to bring content from home into another virtual world, namely OpenSimulator, but on a private server. It would be pretty easy to create an experience very similar to home that way, minus most of the minigames. It will also run on just about anything, so anyone with a windows pc should be able to run it just fine.

**Comment by snakescater** [comment]
That would be so cool, I do have a high-end server laying around somewhere, we could use it for testing if needed.

**Comment by GTPunk** [comment]
@RedFox102 can you work in unity? i already posted my harbor for vrchat zip archive. you need only log in 
 into vrchat settings (sdk already there), upload it and boom! its in the vrchat. if you doesnt know its playable without any vr devices.

**Comment by GTPunk** [comment]
or at least we can replace hex-values.like cheat engine works. rpcs3 says when it asking for online its 0xae1aa0 , if you can find offline mode hex value, (maybe using hex editor or similar) we can replace it in cheat engine

**Comment by MercuryValentine** [comment]
Yes, that was a post I asked a friend to make since he had an account there. I've acquired more data since that post from various people, about 70GB (which is still pathetic, and the vast majority never get around to extracting Home, sadly), but it seems you can only edit/update posts so long.

**Comment by GTPunk** [comment]
is it possible to replace hex values of connecting to sony servers with offline mode (in rpcs3 with cheat engine or similar way)?

**Comment by RedFox102** [comment]
Here's a thought, what if the HDK acted as a private server for "offline" mode? 

I imagine starting in offline mode was done using the HDK, because there certainly don't seem to be any hidden options in the "dev" versions I've tried. But hey, I'm no expert.

Getting our hands on any version of the HDK would be the holy Grail, but I'm not gonna get my hopes up. I mean how many studios total we're working on content for home? Probably not a large number.

**Comment by MercuryValentine** [comment]
[Here](https://www.dropbox.com/sh/xk873x88nwucq29/AABLny-JIo5GlTyvVbjvoYyaa?dl=0) are the two developer versions I've acquired. NPIA00010 is the full developer version, and while NPEA00013 was the closed beta, it seems they reused the ID and made it for content developers after the closed beta ended. Functionally, they seem to be identical as far as I can tell. You can run them on CFW by copying them to your PS3 and rebuilding the database. 


@RedFox102 That would make sense. All the unencrypted SDC files I have are just XMLs with references to local directories and content, which also fits with why only the developer EBOOT referenced an SDC file in offline mode. That would certainly suck, since as you said, the chances of getting the HDK are  small. And having it would make reverse engineering all the filetypes a cinch too.

In terms of numbers, there were actually quite a lot of solo developers and companies that temporarily worked on Home, but obviously they likely wouldn't have it anymore, even if they were generous enough to leak it. I'm surprised Cubehouse/Jamie Holding contributed to this archive, with no lashback from Sony at that.

**Comment by IDelirious** [comment]
@GTPunk .... im farely intrsted in your work... jave u tried recreating any other maps?

**Comment by GTPunk** [comment]
@IDelirious actually i working on new version. here it is https://yadi.sk/d/eOL2O5ah3abmZF there are 'you are not supposed to see this' rock as a chair, and also i planned an menu here.
@MercuryValentine can you export basic character textures for me please? i am already exported model of basic player from four kings, and i think to retexture it to psh one
Edit/Update: also i found in VRChat a guy called Canaha who created 'home square' map, and it looks 100% like original.

**Comment by GTPunk** [comment]
i've released PSHome map at vrchat. its public. there are tv and ps4 from quest for greatness (i combined videos in one to play on it, but sony blocks it in some countries, such as us =(  ).

**Comment by IDelirious** [comment]
@GTPunk thats fantastic!! can you send me the unity project with it inside?..... i wanna see if there is anything i could do to improve on it

**Comment by GTPunk** [comment]
@IDelirious give me your contacts, facebook, steam, mail, something

**Comment by IDelirious** [comment]
my email is 29jhamon@gmail.com ... you can find me there

**Comment by GTPunk** [comment]
@IDelirious check your spam if you doesn't have it

**Comment by GTPunk** [comment]
@MercuryValentine i've tryed to export player files myself with offzip,but this exports like this https://i.imgur.com/BwVshfb.png can u help please?

**Comment by GTPunk** [comment]
@RedFox102 'offline mode' (x,triangle,square ect) is stands for this menu https://i.imgur.com/BPmsk7B.png  idk what comands uses console. maybe its possible to emulate that hubps3.online.scee.com? maybe its possible to view maps thru that cososole? what else is possible with that developer build?

**Comment by RedFox102** [comment]
@GTPunk That's a new one to me, I never tried with a version that old. Does anything actually happen when you open the console? I did have a short list of commands at some point, which I'm sure included something about loading scenes, but I never got to try any of them because I never got the console to open. 

The old developer conferences that are still available online point to the HDK being a requirement though, so I assume you'd just get an error when you try to actually load something in offline mode, even if it's just harbor studio. I think we have a better chance of finding someone who can break the encryption though. X3

**Comment by GTPunk** [comment]
@RedFox102 yeah console opens. but i cant type anything in rpcs3. try just now that developer versions that mercury valentine uploads. maybe you will succeed on those https://i.imgur.com/aY7WGDd.png

**Comment by GTPunk** [comment]
@RedFox102 what console comands do you know? i found this picture https://twitter.com/Pongo86_/status/933307848095879168

**Comment by RedFox102** [comment]
@GTPunk Sorry, I don't remember any of them, nor do I remember where I found them in the first place. Literally all of my knowledge on this stuff is coming from old forum posts, YouTube videos, and the old home dev conference's, so I'm in the same boat as you are.

Seems like you have a pretty good lead on twitter though. One of their other posts mentions using Maya, and I think that there was a plug-in (Extension? Script? I've never used Maya) that was used for exporting the model's for use in home, I wonder if the same plug-in could import the files we have. Maybe you'll get a response from them, let us know. X3

**Comment by snakescater** [comment]
I think the only way now is to find a former content developer who is willing to shed light on this subject. Need to keep looking for individuals who post things about this, and just ask them. There has to be at least one person who can help out.

**Comment by MercuryValentine** [comment]
Some people I know already contacted Pongo86 and tried booting into Home via the console a few months ago, and it just crashes. If I remember correctly the command was just "home". Running it through ProDG didn't give any obvious hints as to what the exact issue was, however Pongo said that it requires a SELF file (which are hosted on DevNet and fingerprinted), so he isn't willing to share it.

If anyone feels like trying their luck, there are quite a lot of commands usually preceded by the string "SimpleCommand" in the Home binaries. I've scanned through NPIA0005 1.00 and 1.86; haven't found anything pertinent to booting offline though.

**Comment by RedFox102** [comment]
That's too bad about Pongo, but I don't blame him a bit, it's not worth ruining a person's life over. At least you got some new information. ^^

Isn't there a command to view individual models? I think I remember seeing something similar on one of the home dev conference videos.

**Comment by GTPunk** [comment]
@MercuryValentine dude since you are actually answering, can you export player textures?

**Comment by MercuryValentine** [comment]
@GTPunk You still haven't gotten offzip to work? The textures should be in OBJECTS.bar, but [here's](https://file.io/uxCKCz) the DDS files I had laying around.

**Comment by GTPunk** [comment]
@MercuryValentine 404 not found
@RedFox102 i found 'map *name of the scene*' comand. but i've tryed some names and it always says no such scene

**Comment by RedFox102** [comment]
@GTPunk For harbor, it should be "BasicApartment".

**Comment by GTPunk** [comment]
@RedFox102 https://i.imgur.com/iUnPLUr.png

**Comment by RedFox102** [comment]
@GTPunk Weird, works for me. I mean, it just freezes like MercuryValentine said, but it doesn't throw an error like that. I also don't have an error about a missing scene list, maybe grab the xml from another version and try again? Also, how are you finding commands?

**Comment by GTPunk** [comment]
@RedFox102 i just remember commands from other games with console.

**Comment by GTPunk** [comment]
@MercuryValentine 404 reupload please

**Comment by MercuryValentine** [comment]
@GTPunk [Link](https://www.dropbox.com/s/20f4qczh6qfqs7x/Clothes.zip?dl=0)

**Comment by GTPunk** [comment]
@MercuryValentine Thank you, there are so many userful stuff!

**Comment by Kobun42** [comment]
I found out the reason why the dev build crashes when using map BasicApartment.
The server can't get the file from the URL. Perhaps if we did something about that, we'd make a bit of progress into getting there

**Comment by snakescater** [comment]
Maybe try uploading the necessary files to a host and using the new url?

**Comment by Kobun42** [comment]
yes, i've thought of that using a local dns at the moment. as of right now i'm attempting to install the dev pkg. i'll see how that turns out.

**Comment by Kobun42** [comment]
Oh, and by the way, for the DEV version, the CACHE folder seems like a mystery to me. It seems undocumented, and that's why I wish we had some insight on how this worked.

**Comment by RedFox102** [comment]
I've also thought of that, but doesn't the XML reference scene files which we don't even have to begin with?

**Comment by Kobun42** [comment]
as far as i'm aware, yes. I'm not sure how we'd obtain those, but I'm trying to work with what I have.

**Comment by RedFox102** [comment]
Well, I assume that the sdat files in the archive are just encrypted scene files, not sure what can be done with them though. You can obviously decrypt them with something like trueancestor, but what you do with the file after that is a mystery to me. I've tried offzip to pull files out of them after decryption, expecting to at least get some textures or something, but I never got anywhere.

**Comment by Kobun42** [comment]
yeah, those files are mysteries to me, and I don't know how they're still not documented.

**Comment by RedFox102** [comment]
I've thought about uploading a couple decrypted dats and posting about them on a game research forum like xentax, from what I've seen it just doesn't look like anyone would be interested.

Maybe something like the littlebigplanet space would attract attention, being that it has exclusive content not seen anywhere else. There are tons of things like that, the infamous space is another popular one that comes to mind.

**Comment by Kobun42** [comment]
well, i wanna take a look at them, if possible, can you link them if possible?

**Comment by MercuryValentine** [comment]
I've ran multiple various Home files (not quite all) through some tests and they're no doubt all encrypted. There was already a thread asking about the .SHARC files on Xentax, but only the forum owner replied. There's quite a number of cross references to the files' strings, including [this](https://www.dropbox.com/s/l4oolnacmnm1s75/Screen%20Shot%202019-03-16%20at%2012.38.51%20PM.png?dl=0) in Home's EBOOT, so static analysis might suffice in getting the decryption key.

**Comment by Kobun42** [comment]
interesting, i'll see what I can do soon.

**Comment by Kobun42** [comment]
As of right now, my current business is to see how the OfflineContent stuff works in the developer version of ps home, because lots of this stuff is unclear, and i'm trying to at least load something. If anyone can provide any details or give me some tips to find it out, that would be appreciated.

**Comment by GTPunk** [comment]
is arcade on vita have something userful besides the 2d home games?

**Comment by Kobun42** [comment]
I think it had some awards, but if that's true, then they're unusable now due to the closure of home.

**Comment by GTPunk** [comment]
@Kobun42 i mean files obviously

**Comment by Kobun42** [comment]
well i'm not sure tbh, never honestly went through it.

**Comment by MercuryValentine** [comment]
@Kobun42 You can try to boot into offline mode by placing a file called HubStartup.txt in your dev_usb0000 port with the line "contentServerOverride http://your.ip.address.here", and enter "home" into the developer console. As I mentioned above though, it simply crashed the developer versions, the 1.83 dev EBOOT doesn't appear to detect it, and Pongo said it requires a fingerprinted SELF file. Perhaps the "map" command would work.

@RedFox102 Do you happen to know if there actually is a command to view models? That would help Kobun42 with decrypting the files, if the encryption keys aren't findable via IDA.

**Comment by GTPunk** [comment]
@MercuryValentine so if i just type my ip address in that txt my basicapartment must automaticly visible by game? i dont understand what i need also to do to just view maps on rpcs3

**Comment by Kobun42** [comment]
Thank you for the offline mode info. I'll investigate what it does.

**Comment by Kobun42** [comment]
Well, I tried the offline mode [especially seeing i'm hosting a custom server via apache] but rpcs3 is the only way I could do it, especially when it can't really install it on my current console [wish to get cfw, but no way for my fw yet] and rpcs3 doesn't have networking capabilities, so it probably would have worked if it had networking. I'm looking into possible alternatives, like the mysterious OfflineContent stuff I've seen.

**Comment by Kobun42** [comment]
I almost got cache working! I just looked at the rpcs3 console to see what it was reading, and i've almost got it!

**Comment by Kobun42** [comment]
only issue is that it brings up a white screen with this error in the dump:
Reading back of partially cached files is unsupported (http://***.***.*.*** [ip has been hidden] /Scenes/PackagedScenes/basicapartment)

**Comment by Kobun42** [comment]
I just got a YOU ARE NOT OLD ENOUGH error! LOL :) We're almost there!

**Comment by Kobun42** [comment]
Oh my... the name loaded before freezing! this is it! Screenshot: http://prntscr.com/mz2pvr

**Comment by GTPunk** [comment]
@Kobun42 woah. its something

**Comment by Kobun42** [comment]
I mean, it still crashes, and leaves me with this rather unhelpful crash log, but still, it is something, after all!
https://pastebin.com/X9ukGJCG

**Comment by GTPunk** [comment]
@Kobun42 looks like its asking for avatar

**Comment by Kobun42** [comment]
huh.. That's something I've yet to find out about. So if anyone knows what I have to do to resolve that error or any leads, please let me know.

**Comment by Kobun42** [comment]
I found this line to raise some questions. "------------------ Assert Message START ------------------
Condition : ms_pConstants!=0L
Message : 
In File ../../HubWorld/HUBAvatar/Avatar.inl at line 75"
I have no idea what this is for, but it's gotta mean something is wrong with the Avatar.

**Comment by RedFox102** [comment]
@MercuryValentine I'm pretty sure there is, but I don't remember what it was. I thought I saw it being used in one of the home developer conference videos, but I skimmed through all of them a while ago and didn't see it.

**Comment by GTPunk** [comment]
...meanwhile, i finished orange theme for my vrc map https://www.deviantart.com/gtpunk/art/Orange-Harbour-Studios-from-PSHome-vrchat-790148096

**Comment by Kobun42** [comment]
@GTPunk Looks very cool. I don't play VRChat anymore, but it looks pretty good to me. [EDIT: I actually wanna try it again thanks to that :)]

On other notes, does anyone have any idea about this part of the crashlog? I know it has something to do with avatar, but it's rather unhelpful. If anyone has any ideas or leads, do let me know, because this is something I need to remotely make it into harbour studio.

"------------------ Assert Message START ------------------
Condition : ms_pConstants!=0L
Message :
In File ../../HubWorld/HUBAvatar/Avatar.inl at line 75"

**Comment by GTPunk** [comment]
@Kobun42 maybe just empty file with same name will do something

**Comment by Kobun42** [comment]
@GTPunk I would do that idea, but this issue arose. The file belongs here: "d:/online/hubbranches/HDK_v121/Code/HubWorld/HUBAvatar/Avatar.inl" If you don't know, this is code for the hdk to be compiled, and we do not have such a thing. There must be something going on.

**Comment by Kobun42** [comment]
I've found multiple files regarding avatar clothes that, for some reason, do not exist yet on my rpcs3 installation. I'll be able to see if they have an impact on what's going on in some way later today.

**Comment by Kobun42** [comment]
So I tried it but to no avail. However, when the time comes for that file to load, it should be fine. Does anyone have any ideas on what to do?

**Comment by GTPunk** [comment]
https://static1.squarespace.com/static/53f79822e4b050d619bb1940/t/54619055e4b0f587f5d119a6/1415680096386/shot873.png?format=1500w so there is a way to load up just only models

**Comment by Kobun42** [comment]
Cool! But how did you get an avatar into there, and everything else, too? Seems interesting.

**Comment by GTPunk** [comment]
@MercuryValentine do you know how to spawn avatar.inl?

**Comment by Kobun42** [comment]
Hold on, I just realized something. RPCS3 does not have network emulation yet, and it may be trying to fetch the BAR file from the server, thinking that it has been obtained, but it hasn't. Until RPCS3 gets networking compatibility or I get something that can run DEX files, I'll have to postpone my search. Hopefully soon 4.84 CFW methods come out soon, because it'd probably make it easier for me.

**Comment by WisppZ** [comment]
Hey, haven't talked here before with you guys but long story short, i have DEX cfw PS3 and i have the dev build with the right scene for BasicApartment and the only problem is loading the avatar, you @GTPunk seemed to get the avatar, but how? This is the final mystery until this is solved!

**Comment by RedFox102** [comment]
Look at the version number in the top right of his screenshot, we don't even have access that build. Would love to know who actually posted the screenshot though.

**Comment by WisppZ** [comment]
I could look for more dev versions, where were these ones got from

**Comment by WisppZ** [comment]
@RedFox102 I will test new method tomorrow, since files here are the files hosted by sony i will download the official copy of ps home and use ps3 proxy server to redirect traffic for example from http://select.ww.home.playstation.net/home/ww/country-select.xml to disk/directory/select.ww.home.playstation.net if you know what i mean, but for every file so it can legit download it and probably i can setup a private server by that

**Comment by GTPunk** [comment]
@WisppZ i just found screenshot from one portfolio

**Comment by WisppZ** [comment]
@GTPunk I have a plan, may get it working today, has to do with proxy server and the home command loading harbour studio...

**Comment by WisppZ** [comment]
Looks like it's rip unless we can somehow get the avatar to load correctly...
1. method would be to place empty file in that directory
2. method would be to get the hdk

**Comment by mh-rb** [comment]
So no one figured out the model thing yet?

I just read through this whole thread, and I believe there is some monumental stuff to get Home working again for us all.

I loved home dearly. I still have the original fat PS3 that I played home on all the time, and it has CFW installed on it still. So if I can be of any help in this development, I would be glad to.

**Comment by MercuryValentine** [comment]
@mh-rb If your fat PS3 still has Home on it, the files on it are most likely unencrypted, meaning you could extract any models on it. It would also help in other ways, so if you're willing to upload it, please do.

Edit: Also, the game folder is in /dev_hdd0/game/NPIA00005

**Comment by WisppZ** [comment]
@mh-rb You would be big help in the development, if you wanna take part, add me on Discord: Wispp#5236

**Comment by Kobun42** [comment]
@mh-rb sounds cool, especially seeing mine can't go to cfw at the moment due to the current version.

Edit: I had earlier said, pre-edit, that I had possibly found some debug pkgs. Well, I did, but they were pretty uneventful and seemed more like QA pkgs than dev pkgs. Better than nothing, I guess.

**Comment by MercuryValentine** [comment]
@Kobun42 Are they 1.21, 1.22, 1.35, and 1.36?

**Comment by mh-rb** [comment]
https://drive.google.com/drive/folders/1CiPv1_NreOKMxdnapoTG5P3r3uh3nsoU?usp=sharing

Hope this can maybe help

**Comment by mh-rb** [comment]
@MercuryValentine https://drive.google.com/drive/folders/1CiPv1_NreOKMxdnapoTG5P3r3uh3nsoU

**Comment by MercuryValentine** [comment]
@mh-rb Thank you! So many people can't be bothered to do something as simple as copying files, it makes my day when someone actually does.

I looked at one of the files, and they are encrypted; on the upside, there might be crossover between the packets Home sends (which are also part encrypted), and the algorithm and/or key used to create the .sharc files from what I've discovered so far.

**Comment by Kobun42** [comment]
@MercuryValentine it appears so. It's really nothing special if we're being honest.

@mh-rb i took a look at your archive, and it seems really nice that you managed to do such a thing!

**Comment by mh-rb** [comment]
Is it anything of use or the same old stuff everyone already has?

**Comment by mh-rb** [comment]
Are you suggesting that I'm inexperienced in how to simply extract a game folder? lol

**Comment by MercuryValentine** [comment]
@mh-rb Me? Lol, I honestly don't know anymore. Of all the people I've asked, at least half couldn't follow instructions significantly more detailed than the ones I gave you. Hell, one ghosted mid-chat.

**Comment by Kobun42** [comment]
@mh-rb no.

**Comment by GTPunk** [comment]
there are software callled 'hex2obj'. is this tool works with .dat?

**Comment by Kobun42** [comment]
I'm not sure about hex2obj, never heard of it.
Other news, I got the NPIA00010 build running on real hardware after installing CFW. I'm still figuring it out, but it looks a lot like an older version of the leaked dev eboot near the end of home's lifespan. I think it's being forced to connect to Sony's server, because it's really not doing anything.

**Comment by GTPunk** [comment]
@kobun42 It's the soft to find model in almost anything thru hex values. but i'm noob in hex. it's on xentax

**Comment by Kobun42** [comment]
i can take a look and see if it can make anything.

**Comment by Puterboy1** [comment]
I am also interested in data-mining the scene files (which are of course .sdat), specifically the Pottermore locations. I want the sound files, so if anyone had accessed them, can you help me obtain them?

**Comment by MercuryValentine** [comment]
@Puterboy1 Have you checked the [ambient-mixer.com](https://www.ambient-mixer.com/) top audio templates?

**Comment by Puterboy1** [comment]
> @Puterboy1 Have you checked the [ambient-mixer.com](https://www.ambient-mixer.com/) top audio templates?

No, I haven’t. Where are they? These templates? Can you direct me to them?

**Comment by Puterboy1** [comment]
> > @Puterboy1 Have you checked the [ambient-mixer.com](https://www.ambient-mixer.com/) top audio templates?
> 
> No, I haven’t. Where are they? These templates? Can you direct me to them?

Well, I found the site. Did the production team behind the Pottermore scenes use these exact sounds for the common room ambiences?

**Comment by GTPunk** [comment]
@MercuryValentine can you try found some models in .dat with hex2obj ? or make script in quickbms?

**Comment by RedFox102** [comment]
@GTPunk It won't work, it's all encrypted. hex2obj was one of the first things I tried.

**Comment by JustNicked00** [comment]
Hello, I'm a fairly big PlayStation Home. Currently 18 years old.
I don't own my PS3 anymore but I'm looking for PlayStation home memories.

With memories I mean the fully working 3D files, the audio files and graphics files.
I already got my hands (or sorry, my mouse ;-) ) on all those Sdat files, put them trough many, many program's. Including every thing I saw on this forum. Is here someone who can help me on completing my collection? I've autism, and I cannot live without this game. Already did that for the past 4 years. I wait for answer.

**Comment by Puterboy1** [comment]
> Hello, I'm a fairly big PlayStation Home. Currently 18 years old.
> I don't own my PS3 anymore but I'm looking for PlayStation home memories.
> 
> With memories I mean the fully working 3D files, the audio files and graphics files.
> I already got my hands (or sorry, my mouse ;-) ) on all those Sdat files, put them trough many, many program's. Including every thing I saw on this forum. Is here someone who can help me on completing my collection? I've autism, and I cannot live without this game. Already did that for the past 4 years. I wait for answer.

Do you have the Slytherin, Ravenclaw and Hufflepuff common rooms?

**Comment by NamelessMasquerade** [comment]
Someone try to contact Pongo86 on twitter ? I think we need some help now ...

**Comment by johndrinkwater** [comment]
> Do you have the Slytherin, Ravenclaw and Hufflepuff common rooms?

If it’s not in this archive, until someone else announces they have files that will be a no. It’s likely some artists that worked on Home may have some assets lying around on backups, but thats going to be a task to get in touch with (and then have the fun of IP issues)
Not sure if @JustNicked00 is suggesting they have more files, or that they downloaded the archive to play with.

**Comment by atomic83GitHub** [comment]
Well, we can now decrypt sdat files with "true ancestor EDAT rebuilder" but the I can't manage to read the .dat file, can you help me to do so?

**Comment by DedSecisHere** [comment]
Hello everyone, 

I hope you guys are still on here and active I found some files u guys might be missing also a bunch of files you might find helpfulI will upload both ps home files the original one and the second one which I used to launch via multiman l. I really wish this comes as helpful to the project as I really want ps home back. I would like to note I used to mod there and I think I have the IGA tool which was similar to dev tool where I did dumb shit with it using a dns . I appreciate your hard work to bring back the game to the community. let me know if you guys need anything else.

zip file with most of mods and my ps home version i had ( https://anonfile.com/m4Zdx916nf/home_files_all_zip )

also found home beta 1.0 and other old pshome files links use wayback machine and i managed to download, you guys can digg further if u want but here is what i found from my research

https://web.archive.org/web/20081026045011/http://www.ps3hax.net/showthread.php?t=7397
http://nopsn.com/pkg/ps3/?dir=playstation-home
https://web.archive.org/web/20091113090336/http://www.ps3news.com/forums/post146-226114.html
tutorial : 
https://www.neowin.net/forum/topic/630195-unofficial-download-home-beta-v10-open-beta-real-soon/
http://select.ww.home.playstation.net/home...duct-select.xml
http://select.ww.home.playstation.net/home...ntry-select.xml
http://pkg.ww.home.playstation.net/home/ww...000000001-E.pkg

**Comment by Puterboy1** [comment]
> Hello everyone,
> 
> I hope you guys are still on here and active I found some files u guys might be missing also a bunch of files you might find helpfulI will upload both ps home files the original one and the second one which I used to launch via multiman l. I really wish this comes as helpful to the project as I really want ps home back. I would like to note I used to mod there and I think I have the IGA tool which was similar to dev tool where I did dumb shit with it using a dns . I appreciate your hard work to bring back the game to the community. let me know if you guys need anything else.
> 
> zip file with most of mods and my ps home version i had ( https://anonfile.com/m4Zdx916nf/home_files_all_zip )
> 
> also found home beta 1.0 and other old pshome files links use wayback machine and i managed to download, you guys can digg further if u want but here is what i found from my research
> 
> https://web.archive.org/web/20081026045011/http://www.ps3hax.net/showthread.php?t=7397
> http://nopsn.com/pkg/ps3/?dir=playstation-home
> https://web.archive.org/web/20091113090336/http://www.ps3news.com/forums/post146-226114.html
> tutorial :
> https://www.neowin.net/forum/topic/630195-unofficial-download-home-beta-v10-open-beta-real-soon/
> http://select.ww.home.playstation.net/home...duct-select.xml
> http://select.ww.home.playstation.net/home...ntry-select.xml
> http://pkg.ww.home.playstation.net/home/ww...000000001-E.pkg

Does it have the missing common rooms from Pottermore?

**Comment by DedSecisHere** [comment]
I'm confused why would you need it ? since home used to run on cache and I think there is a way to make it run without the missing maps . It's worth to meantion home used to run on HDD not cache in early versions, which is what i tried to research and find. Anyways here is a worldmap with some maps that I don't think was in other submissions (common rooms aren't there )

https://anonfile.com/B8F9z61cn2/WORLDMAP_zip

**Comment by DedSecisHere** [comment]
I'm confused why would you need it ? since home used to run on cache and I think there is a way to make it run without the missing maps . It's worth to meantion home used to run on HDD not cache in early versions, which is what i tried to research and find. Anyways here is a worldmap with some maps that I don't think was in other submissions (common rooms aren't there )

https://anonfile.com/B8F9z61cn2/WORLDMAP_zip

**Comment by GTPunk** [comment]
@DedSecisHere , so you're like can run home with CFW or something? i kinda stupid, how i can run at least model viewer or map or something on my rpcs3?

**Comment by DedSecisHere** [comment]
> @DedSecisHere , so you're like can run home with CFW or something? I kinda stupid, how I can run at least model viewer or map or something on my rpcs3?

What I meant is some people say home isn't running because of maps and cache and missing files. I think that's not 100% true as home early versions including the dev ver has the ability to load stuff out off HDD not cache. there is something that I believe is possible to do to make the game run.

**Comment by DedSecisHere** [comment]
I created a discord server for those who want to work on the project please join https://discord.gg/Vz4kV9 if you want to. It would be better if we work and contribute wi each other to get the project going.

**Comment by Gaz492** [comment]
So I have managed to find 2 different versions of a developer version.
One appears to be version 1.41 and the other one shows v01.00.17 [Developers' Build].

The latter allows me to open a in game debug console where i am able to type commands in, however if i try to use a command such as home (should take me to the BasicApparment) it crashes the console. Same thing happens in RPCS3.

The 1.41 version only works on my CFW ps3 and i can only get as far as the Start screen where i get 2 options Go online or change region. I have not been able to find a way of opening a debug console in this version, but the dump i found contains a commands history text file in the USRDIR with some commands that the person who dumped the version tried to use. So i presume there is a debug console somewhere. 

the 1.41 build is located here along with a developer eboot for 1.83 https://archive.org/download/PS3ProtoReupsRedump

**Comment by DedSecisHere** [comment]
Hello,

so here is discord server where everyone is there , if you are interested in the project feel free to join
https://discord.gg/QguSBT3

**Comment by VTSTech** [comment]
We have packet captures now btw.

https://ps3-capture-project.fandom.com/wiki/PlayStation_Home

**Comment by ghost** [comment]
To anyone asking for help with Playstation Home files or having Home data on their PS3s, please join this server so we can extract and use those data in a offline 1.86 build with the HDK 1.86 available: https://discord.gg/p8sYDNb

**Comment by ControVR** [comment]
I know this thread is probably dead and buried, but if someone could give me a rundown (or just extract the models and textures for me) on how to extract the maps from pshome, i'd appreciate it. I'm trying to get Sully's Bar, but i literally have found nothing on how to get the map files. (plus the discord server link is dead, which isn't surprising lol)

**Comment by Kobun42** [comment]
> I know this thread is probably dead and buried, but if someone could give me a rundown (or just extract the models and textures for me) on how to extract the maps from pshome, i'd appreciate it. I'm trying to get Sully's Bar, but i literally have found nothing on how to get the map files. (plus the discord server link is dead, which isn't surprising lol)

There's a special version of Noesis out there that can view and export PSHome models to other formats. I'll have to see if I still have it though.

**Comment by ControVR** [comment]
> > I know this thread is probably dead and buried, but if someone could give me a rundown (or just extract the models and textures for me) on how to extract the maps from pshome, i'd appreciate it. I'm trying to get Sully's Bar, but i literally have found nothing on how to get the map files. (plus the discord server link is dead, which isn't surprising lol)
> 
> There's a special version of Noesis out there that can view and export PSHome models to other formats. I'll have to see if I still have it though.

i found the noesis version, and i managed to get maps imported and exported and into blender. thanks for letting me know it existed lmao

**Comment by neptunedotnet** [comment]
> > > I know this thread is probably dead and buried, but if someone could give me a rundown (or just extract the models and textures for me) on how to extract the maps from pshome, i'd appreciate it. I'm trying to get Sully's Bar, but i literally have found nothing on how to get the map files. (plus the discord server link is dead, which isn't surprising lol)
> > 
> > 
> > There's a special version of Noesis out there that can view and export PSHome models to other formats. I'll have to see if I still have it though.
> 
> i found the noesis version, and i managed to get maps imported and exported and into blender. thanks for letting me know it existed lmao

do you know where i could find this version? i have been looking for it but i can't seem to find it

**Comment by Kobun42** [comment]
> > > > I know this thread is probably dead and buried, but if someone could give me a rundown (or just extract the models and textures for me) on how to extract the maps from pshome, i'd appreciate it. I'm trying to get Sully's Bar, but i literally have found nothing on how to get the map files. (plus the discord server link is dead, which isn't surprising lol)
> > > 
> > > 
> > > There's a special version of Noesis out there that can view and export PSHome models to other formats. I'll have to see if I still have it though.
> > 
> > 
> > i found the noesis version, and i managed to get maps imported and exported and into blender. thanks for letting me know it existed lmao
> 
> do you know where i could find this version? i have been looking for it but i can't seem to find it

I think I have it in a zip file full of other home stuff, I'll see if I can separate it and get it uploaded.
Edit: Done! [You can get it here.](https://drive.google.com/file/d/1zIsZb2hw46Y9gPuAV4KRoTiSkr4U-W6D/view?usp=sharing)

**Comment by neptunedotnet** [comment]
> > > > > I know this thread is probably dead and buried, but if someone could give me a rundown (or just extract the models and textures for me) on how to extract the maps from pshome, i'd appreciate it. I'm trying to get Sully's Bar, but i literally have found nothing on how to get the map files. (plus the discord server link is dead, which isn't surprising lol)
> > > > 
> > > > 
> > > > There's a special version of Noesis out there that can view and export PSHome models to other formats. I'll have to see if I still have it though.
> > > 
> > > 
> > > i found the noesis version, and i managed to get maps imported and exported and into blender. thanks for letting me know it existed lmao
> > 
> > 
> > do you know where i could find this version? i have been looking for it but i can't seem to find it
> 
> I think I have it in a zip file full of other home stuff, I'll see if I can separate it and get it uploaded. Edit: Done! [You can get it here.](https://drive.google.com/file/d/1zIsZb2hw46Y9gPuAV4KRoTiSkr4U-W6D/view?usp=sharing)

is there anything specific i need to do to get it to work? it doesnt seem to let me export or view the files

**Comment by Kobun42** [comment]
> > > > > > I know this thread is probably dead and buried, but if someone could give me a rundown (or just extract the models and textures for me) on how to extract the maps from pshome, i'd appreciate it. I'm trying to get Sully's Bar, but i literally have found nothing on how to get the map files. (plus the discord server link is dead, which isn't surprising lol)
> > > > > 
> > > > > 
> > > > > There's a special version of Noesis out there that can view and export PSHome models to other formats. I'll have to see if I still have it though.
> > > > 
> > > > 
> > > > i found the noesis version, and i managed to get maps imported and exported and into blender. thanks for letting me know it existed lmao
> > > 
> > > 
> > > do you know where i could find this version? i have been looking for it but i can't seem to find it
> > 
> > 
> > I think I have it in a zip file full of other home stuff, I'll see if I can separate it and get it uploaded. Edit: Done! [You can get it here.](https://drive.google.com/file/d/1zIsZb2hw46Y9gPuAV4KRoTiSkr4U-W6D/view?usp=sharing)
> 
> is there anything specific i need to do to get it to work? it doesnt seem to let me export or view the files

You need to have the actual model files (using the .mdl extension) from the scene/object, not the sdat/bar/sharc.

A good tool to extract Home sdats among other things is Nautilus, [which you can find here](https://github.com/DeViL303/MultiServer3-NautilusFork/releases/tag/WIP00026). 
EDIT: As of recently, hdk-cli, [which you can find here](https://github.com/ZephyrCodesStuff/hdk-cli/releases/tag/release-89fb9f2) is also an option if you fancy a Linux-compatible command-line program. I do think Nautilus is easier for the layman, but if you want just the bare essentials, hdk-cli works well enough. Not exactly "endorsing" either of them, just know that both are very nice options if you need to extract an SDAT.

**Comment by DeViL303** [comment]
Some of you might find this tool useful, it can unpack everything related to home. 

https://github.com/DeViL303/MultiServer3-NautilusFork/blob/main/README.md

**Comment by Ocublox** [comment]
Hi, has anyone found out how to load a models skeleton _(.SKN)_ and animation _(.ANI)_ in Noesis? I've been searching for a while now and found nothing. Any information would be appreciated, thanks.

**Comment by Kobun42** [comment]
> Hi, has anyone found out how to load a models skeleton _(.SKN)_ and animation _(.ANI)_ in Noesis? I've been searching for a while now and found nothing. Any information would be appreciated, thanks.

My understanding is that Noesis was largely only ever intended for viewing static .MDL files and it likely won't be updated to add those file formats unless anyone genuinely wants to work on that (then again, considering the PSHome version of Noesis' dicey origins, I doubt that'll happen)

HDK versions of Home do have a built-in Character Viewer or HDK Browser mode that I THINK (not 100% sure, been a long time since I messed with those) might allow you to view them. If you're looking to DUMP them to a different format, however, I don't think there's anything at all that'll help you with that right now unless anyone wants to just straight up make a new tool for that.
